src/client/pages/Embark/index.tsx

```diff
@@ -179,7 +181,10 @@ const Embark = (props: EmbarkProps) => {
                       onClick={handleClickPhoneNumber}
                     />
                   ) : (
-                    <LanguagePicker color="black" />
+                    <LanguagePicker
+                      color="black"
+                      performClientSideNavigation={market !== Market.No}
```

Comment:
Avoid "market checks", they don't scale => I think this case is better solved by a feature flag

Reply:
🤔 what would you call that feature flag?